cocos/core/assets/morph-rendering.ts

```diff
@@ -48,7 +49,8 @@ export class StdMorphRendering implements MorphRendering {
                 continue;
             }
 
-            if (preferCpuComputing) {
+            const hasFeatureTextureFloat = gfxDevice.hasFeature(GFXFeature.TEXTURE_FLOAT);
+            if (preferCpuComputing || !hasFeatureTextureFloat) {
```

Comment:
I think `preferCpuComputing` is not necessary, we should only depend on the feature support to decide whether to use GPU computing. If you think there are other conditions to choose cpu computing, then `preferCpuComputing` should be a configurable API.
By the way, in the current logic, even if the device has texture float support, we are still using cpu computing, there is no way to turn on gpu computing.

Reply:
I have now turned the `preferCpuComputing` to `false`. Cpu rendering would help debugging.